solr/schema.xml

```diff
@@ -75,6 +75,10 @@
    <field name="notes" type="text_general" indexed="true" stored="true" multiValued="false"/>
    <field name="years" type="int" indexed="true" stored="false" multiValued="true"/>
    <field name="year_sort" type="sort_string" indexed="true" stored="false" multiValued="false"/>
+   <field name="dates" type="text_general" indexed="true" stored="true" multiValued="true"/>
+   <field name="extents" type="text_general" indexed="true" stored="true" multiValued="true"/>
+   <field name="context" type="text_general" indexed="true" stored="true" multiValued="true"/>
+   <field name="context_uri" type="string" indexed="true" stored="true" multiValued="true"/>
```

Comment:
@sem42198 These should be removed as well post commit 4d99a35c398cfe871a6d34d4c8b9177ded4d6efa

Reply:
Good catch. I've removed those fields